controllers/activities.js

```diff
@@ -0,0 +1,91 @@
+const models = require('../models')
+
+const getAllActivities = async (request, response) => {
+  try {
+    const activities = await models.Activities.findAll()
+
+    return response.send(activities)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
+
+const getActivitiesByMoon = async (request, response) => {
+  try {
+    const { moonId } = request.params
+
+    const foundMoon = await models.Activities.findOne({
+      where: { moonId }
+    })
+
+    return foundMoon
+      ? response.send(foundMoon)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
+
+const getActivitiesByMonth = async (request, response) => {
+  try {
+    const { monthId } = request.params
+
+    const foundMonth = await models.Activities.findOne({
+      where: { monthId }
+    })
+
+    return foundMonth
+      ? response.send(foundMonth)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
+
+const addActivities = async (request, response) => {
+  try {
+    const { name, moonId, monthId } = request.body
+
+
+    if (!name || !moonId || !monthId) {
+      return response.status(400).send('Please provide requested data.')
+    }
+
+    const [newActivity] = await models.Activities.findOrCreate({ where: { name }, defaults: { moonId, monthId } })
```

Comment:
The weird relationships and duplicate migrations are causing some real problems with your models connecting to the DB correctly which is why this route fails. Note on the get route, you get two monthIds. All that needs some major clean up before any of this will work.

![image](https://user-images.githubusercontent.com/5775716/84072336-05c55700-a99d-11ea-863a-e015aa23a41b.png)